# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/SocketProcessChild.h
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 netwerk/ipc/SocketProcessChild.h
--- a/netwerk/ipc/SocketProcessChild.h	Tue Sep 17 18:15:23 2019 +0000
+++ b/netwerk/ipc/SocketProcessChild.h	Tue Sep 17 18:15:41 2019 +0000
@@ -45,9 +45,9 @@
       Endpoint<mozilla::PProfilerChild>&& aEndpoint);
   mozilla::ipc::IPCResult RecvSocketProcessTelemetryPing();
 
-  PWebrtcProxyChannelChild* AllocPWebrtcProxyChannelChild(
+  PWebrtcTCPSocketChild* AllocPWebrtcTCPSocketChild(
       const PBrowserOrId& browser);
-  bool DeallocPWebrtcProxyChannelChild(PWebrtcProxyChannelChild* aActor);
+  bool DeallocPWebrtcTCPSocketChild(PWebrtcTCPSocketChild* aActor);
   PDNSRequestChild* AllocPDNSRequestChild(
       const nsCString& aHost, const OriginAttributes& aOriginAttributes,
       const uint32_t& aFlags);